CONTRIBUTING.md

```diff
@@ -13,15 +13,15 @@ Contributing code to snap is a snap (pun intended).
 - Push your commit(s) to your repository
 - Open a pull request against the original repo and follow the pull request guidelines below
 
-The maintainers of the repo utilize a "Looks Good To Me" (LGTM) message in the pull request.
+The maintainers of the repo utilize a "Looks Good To Me" (LGTM) message in the pull request. After one or more maintainer states LGTM, we will merge. If you have questions or comments on your code, feel free to correct these in your branch through new commits.
 
 ### Commit Guidelines
 
 Commits should have logical groupings. A bug fix should be a single commit. A new feature
-should be a single commit. 
+should be a single commit.
```

Comment:
Cool, that just means we would also have to be more consistent as maintainers by insisting on single-commit PRs. I hadn't seen many requests to squash before merge, which is why I asked for the clarification. Good thing to explain here.

> On Mar 9, 2016, at 12:05, Matt Brender <notifications@github.com> wrote:
> 
> In CONTRIBUTING.md:
> 
> >  
> >  ### Commit Guidelines
> >  
> >  Commits should have logical groupings. A bug fix should be a single commit. A new feature
> > -should be a single commit. 
> > +should be a single commit.
> Yes, if a PR is opened with 5 commits that was the work involved to fix a single issue, it should be rebased to a single commit. If a PR is opened with 5 commits, with the first three to fix one issue and the second two to fix a separate issue, then it should be rebased to two commits, one for each issue.
> 
> I'd like to elaborate on this section given that example. Thanks for that @geauxvirtual & @ConnorDoyle
> 
> —
> Reply to this email directly or view it on GitHub.
>